### PR TITLE
[Tests-only]  Skeleton files  used wisely on webui sharing and autocomplete feature files

### PR DESCRIPTION
**Pull request by SwikritiT**
##  Description
refactor steps with appropriate skeleton files.
Following suites have been refactored:
- webUISharingAutocompletion
- webUISharingInternalGroups
- webUISharingInternalGroupsEdgeCases
- webUISharingInternalGroupsSharingIndicator

## Related Issue

- Part of owncloud/QA#659

## How Has This Been Tested?

- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13931/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13931/

<p>




</p></details>

**Review by Talank — COMMENTED** [review]
LGTM

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicDifferentRoles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13942/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13942/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13948/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13955/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13957/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicDifferentRoles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13985/

<p>




</p></details>

File: tests/acceptance/features/webUISharingInternalGroupsEdgeCases/shareWithGroupsEdgeCases.feature [hunk 1]
```diff
@@ -10,14 +10,16 @@ Feature: Sharing files and folders with internal groups
 
   @skip @yetToImplement
   Scenario Outline: sharing  files and folder with an internal problematic group name
-    Given these users have been created with default attributes:
+    Given these users have been created with default attributes and without skeleton files:
```
Comment (kiranparajuli589) [inline tests/acceptance/features/webUISharingInternalGroupsEdgeCases/shareWithGroupsEdgeCases.feature:13]: this can go to the background.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14015/

<p>





</p></details>

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm

**Review by individual-it — APPROVED** [review]

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14309/

<p>




</p></details>

**Pull request merged**